torch/nn/modules/linear.py

```diff
@@ -178,4 +179,78 @@ def extra_repr(self) -> str:
             self.in1_features, self.in2_features, self.out_features, self.bias is not None
         )
 
+
+class LazyLinear(LazyModuleMixin, Linear):
+    r"""A :class:`torch.nn.Linear` module with lazy initialization.
+
+    This module will initialize the `weight` and `bias` parameters after
+    the first call to ``forward`` is done. After that the module will
```

Comment:
It'd be interesting to see how the weight and bias are constructed from the input here.

Reply:
I extended the docstring to reflect this